### PR TITLE
refactor: transfer & multipayment in single modal

### PR DESCRIPTION
**Pull request by brenopolanski**
## Summary

- Remove dropdown and multipayment button from the UI.
- Make Multipayment transaction modals the default SEND action for the wallet.
- Implement logic for the wallet to discern type of transaction based on number of recipients entered.
- Transfer modal allows URI links.
- Show the old "send" form (normal transaction) if the AIP11 flag isn't enabled.
- Hide multipayment functionality when using a ledger wallet.

## Checklist

- [ ] Documentation _(if necessary)_
- [x] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

**Comment by alexbarnsley** [comment]
@brenopolanski there may be a nice conflict for you here :trollface: let me know once you're happy for me and Sam to review

**Comment by brenopolanski** [comment]
> @brenopolanski there may be a nice conflict for you here :trollface: let me know once you're happy for me and Sam to review

Done @alexbarnsley

File: src/renderer/components/Transaction/TransactionForm/TransactionFormTransfer.vue [hunk 1]
```diff
@@ -304,8 +304,12 @@ export default {
       return this.form.recipients.length > 1
     },
 
+    isLedgerConnected () {
+      return this.$store.getters['ledger/isConnected']
+    },
+
     isLedger () {
-      return this.currentWallet && !!this.currentWallet.isLedger
+      return this.isLedgerConnected || (this.currentWallet && !!this.currentWallet.isLedger)
```
Comment (dated) [inline src/renderer/components/Transaction/TransactionForm/TransactionFormTransfer.vue:312]: Shouldn't this be `&&`?

Comment (brenopolanski) [reply src/renderer/components/Transaction/TransactionForm/TransactionFormTransfer.vue:312]: yep! Thanks @dated :)

**Pull request merged**